model/mcp/common.yaml

```diff
@@ -0,0 +1,41 @@
+groups:
+  - id: mcp.common.attributes
+    type: attribute_group
+    brief: Common MCP attributes
+    attributes:
+      - ref: mcp.tool.name
+        requirement_level:
+          conditionally_required: When operation is related to a specific tool.
+      - ref: mcp.prompt.name
+        requirement_level:
+          conditionally_required: When operation is related to a specific prompt.
+      - ref: network.transport
+        brief: The transport protocol used for the MCP session.
+        note: |
+          This attribute SHOULD be set to `tcp` or `quic` if the transport protocol
+          is HTTP with SSE (Server-Sent Events).
```

Comment:
@lmolkova the latest spec only have stdio and Streamable HTTP now https://modelcontextprotocol.io/specification/2025-03-26/basic/transports

Reply:
fixed in https://github.com/open-telemetry/semantic-conventions/pull/2083/commits/dc77673926c7b236f62440cf70f1dcc79bebc575